src/atap_widgets/conversation.py

```diff
@@ -334,6 +334,29 @@ def get_topic_recurrence(
 
         return recurrence
 
+    @staticmethod
+    def get_conceptual_recurrence_rate(similarity: pd.DataFrame) -> float:
+        """
+        Calculate the overall conceptual recurrence rate for the conversation.
+
+        See https://doi.org/10.1063/1.5024809
+
+        See `Conversation.get_grouped_recurrence()` for the person-to-person
+        and group-to-group metrics which are related to this
+
+        Args:
+            similarity: A matrix of similarity scores for the conversation.
+
+        Returns:
+            Conceptual recurrence rate, a single value.
+        """
+        similarity_upper = pd.DataFrame(
```

Comment:
Is this just the average of the upper triangle?

Reply:
Looks like it, but I like having the calculation correspond to the calculation in the paper. Made some comments and added this to the tests for completeness.